backend/src/test/java/gov/cdc/usds/simplereport/api/converter/FhirConverterTest.java

```diff
@@ -1273,6 +1282,9 @@ void createFhirBundle_TestEvent_matchesJson() throws IOException {
     expectedSerialized = expectedSerialized.replace("$MESSAGE_HEADER_ID", messageHeaderId);
     expectedSerialized = expectedSerialized.replace("$PRACTITIONER_ROLE_ID", practitionerRoleId);
     expectedSerialized = expectedSerialized.replace("$PROVENANCE_ID", provenanceId);
+    expectedSerialized =
+        expectedSerialized.replace(
+            "$EFFECTIVE_DATE_TIME_TESTED", new DateTimeType(dateTested).getValueAsString());
```

Comment:
non-blocking question: could you hardcode this in this case instead of doing the replace thing?

Reply:
I do not think so 🤔, since this becomes time date time and includes a timezone. 
For example, if I hard code the date to 0 epoch time and then set my timezone somewhere else it fails. 
<img width="321" alt="image" src="https://user-images.githubusercontent.com/10108172/223808549-eb77b51b-6a27-48c5-89f8-b656f5bb34ef.png">